src/locales/fi.json

```diff
@@ -473,7 +468,6 @@
       "alkuPvm": "Hankkeen alkupäivä",
       "loppuPvm": "Hankkeen loppupäivä",
       "tyomaaTyyppi": "Työn tyyppi",
-      "tyomaaKoko": "Hankkeen koko",
```

Comment:
There's also hankePortfolio.labels.tyomaanKoko for each language.

Reply:
Had missed those. Now they are removed as well.